secedgar/utils/cik_map.py

```diff
@@ -1,25 +0,0 @@
-import json
```

Comment:
Why did you delete this file?

Reply:
I moved the CIK Map into the `CIKLookup`.